src/_guides/libraries/library-tour.md

```diff
@@ -2407,6 +2378,11 @@ invoke functions or methods at runtime.
 The dart:mirrors library works in both web apps and command-line apps.
 To use it, import dart:mirrors.
 
+<?code-excerpt "test/library_tour/mirrors_test.dart (import)"?>
+{% prettify dart %}
+import 'dart:mirrors';
+{% endprettify %}
+
 <div class="alert alert-warning" markdown="1">
 **Warning:**
 Using dart:mirrors can cause dart2js to generate very large JavaScript
```

Comment:
I wonder if we are ready to update this section.  Maybe keep mirrors & `@MirrorsUsed` but also add a paragraph about `package:build` etc. Not necessarily in this PR.

cc @kevmoo to see if we're ready for this. Newcomers to Dart still learn about mirrors before anything else.